src/main/java/com/puppycrawl/tools/checkstyle/checks/FinalParametersCheck.java

```diff
@@ -76,6 +89,17 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example :
+ * </p>
+ * <pre>
+ * class MyClass {
+ *   public MyClass(int num) { // violation, num should be final
+ *   }
+ *   public void myMethod(String str) { // OK
```

Comment:
one more time:
the same class (with 2 ctors and one menthod) should be used in all examples.